tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -1703,6 +1702,25 @@ def testDouble(self):
 
 class RoundingTest(tf.test.TestCase):
 
+  def _compare_values(self, x, y, use_gpu):
+    y = np.asarray(y)
+    with self.test_session(use_gpu=use_gpu) as sess:
+      iny = tf.convert_to_tensor(y)
+      inx = tf.convert_to_tensor(x)
```

Comment:
The `iny` and `inx` are unnecessary.  You don't use `iny` at all, and the `inx` conversion will happen automatically.